.github/ISSUE_TEMPLATE/regular.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Regular
```

Comment:
How about "Blank" or "Other" instead of "Regular"? "Regular" suggests to me that most issues should be filed with this template, which I think is not true. (Most should be using one of the other templates)

Reply:
Yes, I think "Other" is better.